layouts/404.html

```diff
@@ -1,29 +1,14 @@
 {{ define "main" }}
 {{ partial "handle-version-switch-404.html" . }}
-<div class="container-404">
-    <h4>
-        Perhaps you were looking for:
-    </h4>
-
-    <br />
-    <br />
-
-    <ul>
-        <li>
-            <a href="/docs">
-                The Kubernetes documentation
-            </a>
-        </li>
-        <li>
-            <a href="/blog">
-                The Kubernetes blog
-            </a>
-        </li>
-        <li>
-            <a href="/community">
-                The Kubernetes community
-            </a>
-        </li>
+<section id="error-404" class="error-details">
+    <p>
+    {{ T "error_404_were_you_looking_for" }}
+    </p>
+    <ul id="error-sections">
+        {{ $sections := slice "docs" "blog" "training" "partners" "community" "case-studies" }}
+        {{ range $sections }}
+        {{ with site.GetPage "section" . }}<li><a href="{{ .RelPermalink }}" data-proofer-ignore>{{ .Title }}</a></li>{{ end }}
```

Comment:
Thanks @sftim. These changes look nice.
One question: At this time, is there a good reason to add "data-proofer-ignore"?